sync2jira/upstream_issue.py

```diff
@@ -358,6 +383,22 @@ def github_issues(upstream, config):
         if issue.get('milestone', None):
             issue['milestone'] = issue['milestone']['title']
 
+        if not issue.get('storypoints', None):
+            issue['storypoints'] = ''
+            orgname, reponame = upstream.rsplit('/', 1)
+            issuenumber = issue['number']
+            github_project_fields = config['sync2jira']['map']['github'][upstream]['github_project_fields']
+            variables = {"orgname": orgname, "reponame": reponame, "issuenumber": issuenumber}
+            for fieldname, values in github_project_fields.items():
+                variables['ghfieldname'] = values[0]
+                response = requests.post(graphqlurl, headers=headers, json={"query": project_items_query, "variables": variables})
+                data = response.json()
+                if fieldname == 'storypoints':
+                    try:
+                        issue[fieldname] = data['data']['repository']['issue']['projectItems']['nodes'][0]['fieldValueByName']['number']
+                    except (TypeError, KeyError):
+                        continue
```

Comment:
We definitely need a log statement here (warning, or error, or.. something else?)

There are so many key-accesses and that one zero-index access that this will surely fail at some point. It will be good to have a clue as to why in the logs.

Reply:
Done. b921b55434d2599f3c7462d25f84c52fde66f305